set-based-validation/README.md

```diff
@@ -0,0 +1,28 @@
+# Set Based Consistency Validation
+A type of question that occurs a lot is "How can I check if an account with a particular email address already exists?". 
+In essence this signals the requirement to be able to validate a set, which is a more intricate process when it comes to applications which follow the CQRS paradigm. Without going in full detail here why this is a problem, a straightforward solution is to enhance the command model by adding a small look-up table which can be queried during such a validation step.
+
+There are several ways to invoke this set validation when using Axon, several of which are shared in this repository.
+
+Prior to checking if the email address already exists you need to save all the email addresses that are used. You can do this by using an event handling component:
+
+https://github.com/AxonIQ/code-samples/blob/set-validation/set-based-validation/src/main/java/io/axoniq/dev/samples/command/handler/AccountEventHandler.java
```

Comment:
I think this link will bread as soon as you delete the `set-validation` branch so you can probably already use `master` as it will work after merged.